pybossa/view/projects.py

```diff
@@ -1697,6 +1696,28 @@ def respond():
 
         return handle_content_type(data)
 
+    def get_users_fullname(page_tasks, get_users_func, result_field):
```

Comment:
The function name "get_users_fullname" seems misleading. This function modifies each task in the "page_tasks" according to "get_users_func" and it doesn't need to return anything. Shall we give a more meaningful name?